code/game/objects/items/shooting_range.dm

```diff
@@ -4,41 +4,16 @@
 	icon = 'icons/obj/objects.dmi'
 	icon_state = "target_h"
 	density = FALSE
+	vis_flags = VIS_INHERIT_ID
 	var/hp = 1800
-	var/obj/structure/target_stake/pinnedLoc
-
-/obj/item/target/Destroy()
-	removeOverlays()
-	if(pinnedLoc)
-		pinnedLoc.nullPinnedTarget()
-	return ..()
-
-/obj/item/target/proc/nullPinnedLoc()
-	pinnedLoc = null
-	density = FALSE
-
-/obj/item/target/proc/removeOverlays()
-	cut_overlays()
-
-/obj/item/target/Move()
-	. = ..()
-	if(pinnedLoc)
-		pinnedLoc.forceMove(loc)
 
 /obj/item/target/welder_act(mob/living/user, obj/item/I)
 	..()
 	if(I.use_tool(src, user, 0, volume=40))
-		removeOverlays()
+		cut_overlays()
```

Comment:
Can you only remove the relevant overlays here?

Reply:
ok